### PR TITLE
Fix for discoverable resource access

### PR DESCRIPTION
**Pull request by pkdash**

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by pkdash** [comment]
@hyi Can you review this bug fix? This needs to go into the next release.

File: hs_core/models.py [hunk 1]
```diff
@@ -103,6 +103,8 @@ def can_view(self, request):
         # have to do import locally to avoid circular import
         from hs_core.views.utils import authorize, ACTION_TO_AUTHORIZE
         return authorize(request, self.short_id, needed_permission=ACTION_TO_AUTHORIZE.VIEW_RESOURCE,
+                         raises_exception=False)[1] or \
+               authorize(request, self.short_id, needed_permission=ACTION_TO_AUTHORIZE.VIEW_METADATA,
```
Comment (hyi) [inline hs_core/models.py]: @pkdash I think authorize() with needed_permission being ACTION_TO_AUTHORIZE.VIEW_METADATA alone is sufficient here, so "or" is not needed.

Comment (pkdash) [inline hs_core/models.py]: @hyi Thanks for catching that.

**Comment by pkdash** [comment]
@hyi See if you can +1

**Comment by hyi** [comment]
+1

**Pull request merged**